src/View/View.php

```diff
@@ -1551,4 +1551,14 @@ protected function _renderElement($file, $data, $options)
 
         return $element;
     }
+
+    /**
+     * Return empty string when View instance is cast to string.
+     *
+     * @return string
+     */
+    public function __toString()
+    {
+        return '';
```

Comment:
Intuitively, I would expect it to internally generate() here really. Rendering the view as is as a whole. This quick-fix for a plugin that did it wrong in the first place sounds misplaced at this point.

Reply:
You probably mean render(). I agree it does seem more intuitive, that's what we do for cells too.